gpu-operator/gpu-driver-configuration.rst

```diff
@@ -217,8 +224,9 @@ The following table describes some of the fields in the custom resource.
      - Specifies the container registry that contains the driver container.
      - ``nvcr.io/nvidia``
 
-   * - ``useOpenKernelModules``
-     - Specifies to use the NVIDIA Open GPU Kernel modules.
+   * - ``useOpenKernelModules`` Deprecated.
+     - This field is depreacted as of v25.3.0. Use ``kernelModuleType`` instead. 
```

Comment:
```suggestion
     - This field is deprecated as of v25.3.0. Use ``kernelModuleType`` instead. 
```

Reply:
Let's clarify that this field will also be ignored by the gpu-operator as of v25.3.0

Reply:
updated!